strawberry/relay/fields.py

```diff
@@ -23,6 +23,7 @@
     Tuple,
     Type,
     Union,
+    UnionType,
```

Comment:
`UnionType` should be imported from `types`, but it is only available on python 3.10+

see `is_union` in `strawberry/utils/typing.py`